Formula/p/passt.rb

```diff
@@ -0,0 +1,31 @@
+class Passt < Formula
+  desc "User-mode networking daemons for virtual machines and namespaces"
+  homepage "https://passt.top/passt/about/"
+  url "https://passt.top/passt/snapshot/passt-2024_07_26.57a21d2.tar.xz"
+  version "2024_07_26.57a21d2"
+  sha256 "8237f5af399ebd49a6946954dc4cc2cbca49c69bfa33a0cf39ac5cfd04d9ddde"
+  license all_of: ["GPL-2.0-or-later", "BSD-3-Clause"]
+  head "git://passt.top/passt", branch: "master"
+
+  depends_on :linux
+
+  def install
+    args = ["prefix=#{prefix}"]
+    args << "VERSION=#{version}" if build.stable?
+    system "make", "install", *args
```

Comment:
These paths need to be set.

```suggestion
    system "make", "install",
      "DESTDIR=#{prefix}/",
      "bindir=bin",
      "man1dir=share/man",
      "docdir=share/doc"
```